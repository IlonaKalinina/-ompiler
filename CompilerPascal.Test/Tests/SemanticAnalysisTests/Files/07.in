program Test_07;
type
people = record
people : integer;
name : string;
end;
var
name : people;
begin
end.