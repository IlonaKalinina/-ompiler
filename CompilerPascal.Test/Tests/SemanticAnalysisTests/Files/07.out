program Test_07
├───type
│    └─── =
│         ├─── people
│         └─── record 
│              ├─── people
│              │    └─── integer
│              └─── name
│                   └─── string
├───var
│    └─── :
│         ├─── name
│         └─── people
│              └─── record 
│                   ├─── people
│                   │    └─── integer
│                   └─── name
│                        └─── string
│    
begin
└─── end

Symbol Tables:
builtins:
	integer: SymInteger
	real: SymReal
	string: SymString
	write: SymProc
	read: SymProc
globals:
	people: SymRecord
		record "people": 
		people: SymbolVar
		name: SymbolVar
	name: SymbolVar
